twine/utils.py

```diff
@@ -243,7 +245,10 @@ def get_userpass_value(
 
 
 get_cacert = functools.partial(get_userpass_value, key="ca_cert")
+get_cacert.__doc__ = "Get the CA certificate via ``get_userpass_value``."
```

Comment:
I believe there's a better way to do this, specifically, there's a `#:` syntax for sphinx so you could do

```py
#: Get the CA certificate via ``get_userpass_value``.
get_cacert = functools.partial(get_userpass_value, key="ca_cert")
```

To achieve the same result without having to access a dunder. I'm not sure this is a useful docstring though. Would it be better to say: 

```rst
Prompt the user for the CA bundle via :func:`get_userpass_value`.
```

Reply:
Thanks for the tip on the `#:` syntax; can you provide a link to the Sphinx docs for that?

Re: "Prompt" vs. "Get", Twine might not actually prompt for the value, if it's coming from keyring.

Reply:
I don't know where that's documented in the Sphinx docs but it's been a supported syntax for close to a decade and works especially well with instance attributes.

Reply:
> can you provide a link to the Sphinx docs for that?

It should be in this section:
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoproperty

Reply:
Thanks for the suggestions. I've updated #811 accordingly, and rebased this PR to include that change.